drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -366,14 +369,21 @@ def can_see_client_details?(user)
             client["#{variant_name}_#{parts[:key]}"] = value
             next unless value
 
+            # Use the previously calculated household_type, for now, just get the first for the client
+            # that matches one of the prior calculations
+            project_client = project_clients[client_id].detect do |pc|
```

Comment:
Is this safe? Should it be
```suggestion
            project_client = project_clients[client_id]&.detect do |pc|
```

Reply:
Well, it _should_ be safe, but probably worth catching.